packages/stark-build/config/webpack-partial.dev.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+const helpers = require("./helpers");
+const buildUtils = require("./build-utils");
+const webpackMerge = require("webpack-merge"); // used to merge webpack configs
+const commonConfig = require("./webpack-partial.common.js"); // the settings that are common to prod and dev
+
+// Dev custom config
+const webpackCustomConfig = require(helpers.root("config/webpack-custom-config.dev.json"));
+
+const isProd = helpers.hasProcessFlag("prod=true");
+
+const METADATA = Object.assign({}, buildUtils.DEFAULT_METADATA, {
+	HOST: process.env.HOST || "localhost",
+	PORT: process.env.PORT || 4200,
+	ENV: isProd ? "production" : "development",
+	environment: helpers.hasProcessFlag("hmr") ? "hmr" : "dev",
+	WATCH: !helpers.hasProcessFlag("watch=false"),
```

Comment:
Why keeping this logic in `build-utils.js` and repeating it here ?

Reply:
True, I'll remove it from here